Code/Tools/ProjectManager/Source/PythonBindingsInterface.h

```diff
@@ -217,11 +217,18 @@ namespace O3DE::ProjectManager
          */
         virtual AZ::Outcome<QVector<GemRepoInfo>, AZStd::string> GetAllGemRepoInfos() = 0;
 
+        /**
+         * Gathers all gem infos for repos
```

Comment:
```suggestion
         * Gathers all gem infos from the designated repo
```

Reply:
some other possible names include...
`GetGemInfosForAllRepos` 
`GetGemInfosForRepo`

Reply:
I missed these suggestions so they'll show up in another PR